### PR TITLE
exported all components;

### LINKED ISSUE #43: Please export all built components
I'm trying to use `DevelopedBy` but it's not exported in the module (I can see the component in the repo though).

Using module ver. `0.1.2`

### PR DESCRIPTION
**Pull request by victorbojica**
Closes #43

### CODE REVIEW

**Comment by victorbojica** [comment]
> I think we have another plugin that deals with svgs called **babel-plugin-inline-react-svg**. Can you take a look to see if we can remove it?

done

File: package.json [hunk 1]
```diff
@@ -28,9 +28,7 @@
     }
   },
   "babel": {
-    "plugins": [
-      "inline-react-svg"
-    ],
+    "plugins": [],
```
Comment (moonflare) [inline package.json]: We will need this one. If you check some components like [ListItem](https://taskforce-fe-compon-git-fork-victorbojica-fix-export-com-e148c8.code4ro.now.sh/?path=/story/listitem--with-inactive-state), it breaks. This module is not doing the same thing. `@rollup/plugin-image` takes care of exporting/importing and `inline-react-svg` makes sure when we use `SVGs` inline they are valid elements. We need these so we can change colors of those svgs, for example. Can you please revert that commit? 🙌

**Comment by moonflare** [comment]
@victorbojica Can you also please bump the version with one patch version? We will need to release this one right after 🚀

File: package.json (hunk 1 above)
Comment (Utwo) [reply package.json]: You are right, my bad, sorry! :)

**Pull request merged**